build/rules.mak

```diff
@@ -61,7 +61,6 @@ $(MODULE_DIR)/%: %.mako $(BUILD_HELPER_SCRIPTS) $(METADATA_FILES)
 
 $(MODULE_DIR)/%_pb2.py: %.proto
 	$(call trace_to_console, "Generating",$@ and $(notdir $*)_pb2_grpc.py)
-	$(_hide_cmds)$(call log_command,mkdir -p $(OUTPUT_DIR)/build/proto/$(dir $*))
```

Comment:
Was this just cruft, leftover from other changes?

Reply:
Yes